FastSimulation/Calorimetry/src/KKCorrectionFactors.cc

```diff
@@ -0,0 +1,75 @@
+#include "FastSimulation/Calorimetry/interface/KKCorrectionFactors.h"
+
+KKCorrectionFactors::KKCorrectionFactors( const edm::ParameterSet& pset )
+{
+
+  interpolate_ = pset.exists("interpolate") && pset.getUntrackedParameter<bool>("interpolate");
```

Comment:
you interpolate anyway,
so please call this option e.g. interpolate3D for clarity